extra/docs_landing.html

```diff
@@ -39,347 +39,607 @@
+          <div class="card h-100">
+            <div class="card-border card-border-top"></div>
+            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltd1f950b35ad63d86/619587857b57030e268d84c9/icon-integrations-color-32.svg');"></span>
+            <h3 class="mt-3">
+              Integrations
+            </h3>
+            <p>Integrations enable you to collect and connect your data with the Elastic Stack.</p>
+          </div>
+        </a>
       </div>
-    </a>
-  </div>
-    <div class="col-md-4 col-12 mb-2">
-      <a class="no-text-decoration" href="https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/getting-started-general-purpose.html">
-        <div class="card h-100">
-          <h4 class="mt-3">
-            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltcb0e79820c355263/636c1905cbd6b84ecb4851b2/stack-logo-color-32px.png');"></span>
-            Custom solution
-          </h4>
-          <p>Deploy your own platform to store, search, and visualize any data.</p>
-        </div>
-      </a>
     </div>
   </div>
 
-  <div class="my-4">
-    <div class="d-flex align-items-center mb-3">
-      <h4 class="mt-3">
-        <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltfb840f627b5923e7/636c1b62358231185a7a8c11/featured-topics-logo-color-32px.png');"></span>
-        Featured topics
-      </h4>
+  <div role="tabpanel" id="deployment-tab-contents" aria-labelledby="deployment-tab" tabindex="0" hidden="">
+    <div class="row">
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/cloud/current/ec-getting-started.html">
+          <div class="card h-100">
+            <div class="card-border card-border-top"></div>
+            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt0dc498ca4c8b3f95/5d104bbf561b9b0b537f9906/logo-cloud-32-color.svg"');">
+            </span>
+            <h3 class="mt-3">
+              Elastic Cloud
+            </h3>
+            <p>Deploy instances of the Elastic Stack in the cloud, with the provider of your choice.</p>
+          </div>
+        </a>
+      </div>
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/elastic-stack/current/installing-elastic-stack.html">
+          <div class="card h-100">
+            <div class="card-border card-border-top"></div>
+            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltcb0e79820c355263/636c1905cbd6b84ecb4851b2/stack-logo-color-32px.png');">
```

Comment:
```suggestion
            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt0090c6239e64faf8/logo-stack-32-color.svg');">
```

Logo is a little hazy, this SVG might be be a good substitute.

Reply:
Nice catch! Fixed here https://github.com/elastic/docs/pull/2847/commits/716d899ee8c266abf80016d2d67650b7e57f6816